src/components/Articles/index.tsx

```diff
@@ -0,0 +1,54 @@
+import styled from 'styled-components'
+import { GetArticlesOutput } from '@/hooks/queries/useGetArticles'
+import { Some } from 'fp-ts/Option'
+import { ArticleCard, EmptyState } from '@/components'
+import format from 'date-fns/format'
+//eslint-disable-next-line @typescript-eslint/no-var-requires
+const readingTime = require('reading-time/lib/reading-time')
+
+const Wrapper = styled.div`
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows: 1fr 1fr 1fr;
+  align-items: center;
+  justify-content: center;
+  gap: 32px 30px;
+`
+
+type ArticlesProps = {
+  articles: Some<GetArticlesOutput>
+}
+
+export const Articles = ({ articles }: ArticlesProps) => {
+  const getReadingTime = (articleBody: string) => {
+    const stats = readingTime(articleBody)
+    const minute = stats.text.charAt(0)
+    return `${minute}min`
+  }
+
+  const getDate = (articleDate: string) => {
+    const formatMethod = 'PP'
+    const date = format(new Date(articleDate), formatMethod)
+    return date
+  }
+
+  return articles.value.articles.length === 0 ? (
```

Comment:
Porque não usar if + return nesse caso? Vai ficar bem mais legível.

Reply:
I fix this in this [commit](https://github.com/joaooliveiradev/conduit-frontend/pull/39/commits/ee7689e6b3935fc2ae24a548bab949263db558dd)